### PR TITLE
Update Reorg

### PR DESCRIPTION
**Pull request by Visorask**
- Added: VF to must not contain to get rid of French subtitle releases.
- Added: [Anime Land] and [Skymoon Raws]  to must not contain as they do not provide subs.
- Changed: Restructured guide to have Optional and Low Quality Releases sections.

Will commit this when verified to not screw up any existing tools.

### CODE REVIEW

File: docs/Sonarr/V3/Sonarr-Release-Profile-RegEx-Anime.md [hunk 1]
```diff
@@ -178,11 +178,9 @@ Add this to your Preferred with a score of **[0]**. This is for renames only and
 /(dual[ ._-]?audio)/i
 ```
 
-#### Optional (uncut|unrated|uncensored)
+#### Uncut|Unrated|Uncensored
```
Comment (rcdailey) [inline docs/Sonarr/V3/Sonarr-Release-Profile-RegEx-Anime.md:181]: Nit: I would just say "Uncensored" here. uncut/unrated aren't really terms I see used in the anime community. This isn't a tooling requirement, just a human readability concern.

File: docs/Sonarr/V3/Sonarr-Release-Profile-RegEx-Anime.md [hunk 2]
```diff
@@ -215,16 +213,17 @@ The reason most of these are added is due to their shitty quality or just in gen
 /(CuaP|PnPSubs|ICEBLUE|SLAX|U3-Web)/i,
 /(Raws-Maji|\bKRP\b|M@nI|Kanjouteki|PuyaSubs)/i,
 /\b(Beatrice|ohys|Kawaiika|neko|daddy)[ ._-]?(raws)\b/i,
-/\b(LowPower|Scryous)[ ._-]?(raws)\b/i,
+/\b(LowPower|Scryous|Skymoon)[ ._-]?(raws)\b/i,
 /\b(NS|AREY|BDMV|BDVD|BJX|DKB|DP|TnF)\b/i,
-/(Amb3r|DsunS|ExREN|\$tore-Chill)/i,
+/(Amb3r|DsunS|ExREN|\$tore-Chill|\bVF\b)/i,
```
Comment (rcdailey) [inline docs/Sonarr/V3/Sonarr-Release-Profile-RegEx-Anime.md:218]: Nit: Group this with VOSFR for housekeeping purposes?

**Review by rcdailey — APPROVED** [review]
Looks great, thank you for the work. I will also close my pull request in favor of yours. All of my comments here are just nitpick, so feel free to ignore them. Just passing some thoughts along to you as I was reviewing the diff.

File: docs/Sonarr/V3/Sonarr-Release-Profile-RegEx-Anime.md (hunk 2 above)
Comment (Visorask) [reply docs/Sonarr/V3/Sonarr-Release-Profile-RegEx-Anime.md:218]: I will reorganize this to more formatted soon but at the moment I try to keep them on one line for Sonarr readability. And that line is full at the moment. It is on my to-do list though to reorder things and have them placed together.

**Comment by TRaSH-** [comment]
Closed because of no updates after 3 months,
re-open when needed.

**Pull request closed without merging**